### PR TITLE
Add transaction to build done

### PR DESCRIPTION
**Pull request by AvoidMe**

### CODE REVIEW

**Review by Korulag — APPROVED** [review]

File: alws/crud/build_node.py [hunk 1]
```diff
@@ -433,29 +440,32 @@ async def __update_built_srpm_url(db: Session, build_task: models.BuildTask):
...
 
-async def build_done(
-            db: Session,
-            request: build_node_schema.BuildDone
-        ):
-    remove_dep_query = delete(models.BuildTaskDependency).where(
-        models.BuildTaskDependency.c.build_task_dependency == request.task_id
-    )
+async def safe_build_done(db: Session, request: build_node_schema.BuildDone):
+    sucess = True
```
Comment (anfimovdm) [inline alws/crud/build_node.py:445]: typo

File: alws/crud/build_node.py [hunk 2]
```diff
@@ -433,29 +440,32 @@ async def __update_built_srpm_url(db: Session, build_task: models.BuildTask):
...
     try:
-        build_task = await __process_build_task_artifacts(
-            db, request.task_id, request.artifacts)
-    except (ArtifactConversionError, ModuleUpdateError, RepositoryAddError) as e:
+        async with db.begin():
+            await build_done(db, request)
+    except Exception:
+        logging.exception('Build done failed:')
+        sucess = False
```
Comment (anfimovdm) [inline alws/crud/build_node.py:451]: same above

File: alws/crud/build_node.py [hunk 3]
```diff
@@ -433,29 +440,32 @@ async def __update_built_srpm_url(db: Session, build_task: models.BuildTask):
...
         await db.execute(update_query)
-
+    finally:
+        remove_dep_query = delete(models.BuildTaskDependency).where(
+            models.BuildTaskDependency.c.build_task_dependency == request.task_id
+        )
         await db.execute(remove_dep_query)
         await db.commit()
-        raise e
+    return sucess
```
Comment (anfimovdm) [inline alws/crud/build_node.py:463]: same above

**Review by anfimovdm — CHANGES_REQUESTED** [review]

**Review by anfimovdm — APPROVED** [review]

**Review by Korulag — APPROVED** [review]

File: alws/constants.py [hunk 4]
```diff
@@ -10,7 +10,7 @@
 
 
 REQUEST_TIMEOUT = 60  # 1 minute
-DRAMATIQ_TASK_TIMEOUT = 3600000  # 1 hour in milliseconds
+DRAMATIQ_TASK_TIMEOUT = 36000000  # 10 hours in milliseconds
```
Comment (soksanichenko) [inline alws/constants.py:13]: You can just write as `60*60*1000`

**Review by soksanichenko — APPROVED** [review]

**Review by Korulag — APPROVED** [review]

**Review by anfimovdm — APPROVED** [review]

**Pull request merged**